src/components/standalone/monitoring/NetifyInformaticsContent.vue

```diff
@@ -0,0 +1,188 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { ref, onMounted, watch } from 'vue'
+import type { Ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import FormLayout from '@/components/standalone/FormLayout.vue'
+import { useNotificationsStore } from '@/stores/notifications'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { NeButton } from '@nethesis/vue-components'
+import {
+  NeSkeleton,
+  NeFormItemLabel,
+  NeTextInput,
+  getAxiosErrorMessage,
+  NeInlineNotification,
+  NeBadge
+} from '@nethesis/vue-components'
+import { NeToggle } from '@nethesis/vue-components'
+import { ubusCall } from '@/lib/standalone/ubus'
+import { AxiosError } from 'axios'
+
+const notificationsStore = useNotificationsStore()
+const { t } = useI18n()
+
+interface Form {
+  status: boolean
+  uuid: string
+}
+
+const form = ref<Form>({
+  status: false,
+  uuid: ''
+})
+
+interface ErrorObject {
+  notificationTitle: string
+  notificationDescription: string
+  uuid: string
+}
+const objError: ErrorObject = {
+  notificationTitle: '',
+  notificationDescription: '',
+  uuid: ''
+}
+
+let isError = ref(false)
+let loading = ref(false)
+let saving = ref(false)
+let uuidRef = ref()
+let JustCopied = ref(false)
+
+const error: Ref<ErrorObject> = ref({ ...objError })
+const errorLoadingConfiguration: Ref<ErrorObject> = ref({ ...objError })
+const errorSaving: Ref<ErrorObject> = ref({ ...objError })
+
+onMounted(() => {
+  getConfiguration()
+})
+
+async function getConfiguration() {
+  try {
+    loading.value = true
+    let getDataConfiguration = await ubusCall('ns.netifyd', 'status', {})
+    if (getDataConfiguration && getDataConfiguration.data) {
+      let configuration = getDataConfiguration.data
+      form.value.status = configuration.enabled
+      form.value.uuid = configuration.uuid
+      loading.value = false
+    }
+  } catch (exception: any) {
+    isError.value = true
```

Comment:
Maybe is better to define the `error` object as nullable, and check for nullable if there's an error set, otherwise we assume state is good

Reply:
happy to do with another way tommaso, but it is a common way to set to true when we have an error, what do you think @andre8244

Reply:
Yes, but in this case the error information is decoupled, during coding if `isError` is not set during an error no message is displayed even if the error is thrown and saved. 
A solution can be of the following:
```ts
const error: Ref<ErrorObject?> = ref()
```
This tells the code that the error can be `undefined` and you can check for it

Reply:
Let's find the quickest solution, but IMHO there are too many variables and typescript types for a simple form with only a toggle :confused: If you got stuck I suggest to use the same variables pattern of other pages (e.g. NatHelpers.vue is a recent and simple page)